tests/acceptance/features/lib/OwncloudPage.php

```diff
@@ -433,6 +433,33 @@ public function getWindowHeight($session) {
 		);
 	}
 
+	/**
+	 * Waits for jquery and the document to load
+	 *
+	 * @param Session $session
+	 * @param int $timeout_msec
+	 *
+	 * @return void
+	 */
+	public function waitForJqueryToLoad(
```

Comment:
In the spirit of the naming convention, should this be following, or is it okay?
```suggestion
	public function waitTillJqueryIsLoaded(
```